### ISSUE TITLE
No way to search for a track without any markets.

### ISSUE THREAD

**Issue by internetztube** [spotify/web-api#1380]
Is there a way to assemble an API search request so that I can get tracks back that aren't in any market?
This track is e.g. not assigned to any market. I could only find it with a Google search.

This would be very helpful!
Thank you.

https://open.spotify.com/track/7j6lQszU5iRu3KJGEn5YWA
https://api.spotify.com/v1/search?q=Endlich%20Nichtschwimmer&type=track

**Comment by mklaunzer** [comment]
Could you please provide some more background regarding the use case you are trying to address with tracks which are not available in any markets?

**Comment by internetztube** [comment]
I have a Title Artist Combo and I need the ISRC. According to my research Spotify is one of the APIs which has a very complete set of ISRCs.

**Comment by internetztube** [comment]
Do you need any further information from me? @mklaunzer

**Comment by mklaunzer** [comment]
Thanks for the infos regarding your use case @internetztube .
Are you specifically looking for tracks which are not available in any market? Otherwise, the market parameter on the search endpoint is optional.

**Comment by internetztube** [comment]
Correct. The market GET parameter is not present on my request. But the result is empty. 

I just want to get back all tracks that fit this search intent. I don't care about the market. The problem is that tracks that are not assigned to a market never appear in its search result.

It is clear to me that no user can play this song. As I said, I want the ISRC.

@mklaunzer 

```
curl -X "GET" "https://api.spotify.com/v1/search?q=Endlich%20Nichtschwimmer&type=track" -H "Accept: application/json" -H "Content-Type: application/json" -H "Authorization: Bearer TOKEN"
```
```
{
  "tracks": {
    "href": "https://api.spotify.com/v1/search?query=Endlich+Nichtschwimmer&type=track&offset=0&limit=20",
    "items": [],
    "limit": 20,
    "next": null,
    "offset": 0,
    "previous": null,
    "total": 0
  }
}
```

**Comment by mklaunzer** [comment]
Understand, thanks for the infos.

We are aiming to provide data to build great consumer experiences. Could you please provide some more information how your use case relates to this goal?
